account_payment_approve_invoice/model/account_invoice.py

```diff
@@ -37,12 +37,12 @@ class AccountInvoice(osv.osv):
         'to_pay': False,
     }
 
-    def copy(self, cr, uid, id, default=None, context=None):
+    def copy(self, default=None):
```

Comment:
¿Where is `api.one`?
¿Where is `.. import api`?

Can you add a unitest in each module to check that copy method work fine!